controllers/perconaservermysql_controller.go

```diff
@@ -143,7 +144,27 @@ func (r *PerconaServerMySQLReconciler) doReconcile(
 	if err := r.cleanupOutdated(ctx, cr); err != nil {
 		return errors.Wrap(err, "cleanup outdated")
 	}
+	if err := r.ensureOperatorVersion(ctx, cr); err != nil {
+		return errors.Wrap(err, "ensure operator version")
+	}
+
+	return nil
+}
+
+func (r *PerconaServerMySQLReconciler) ensureOperatorVersion(ctx context.Context, cr *apiv1alpha1.PerconaServerMySQL) error {
```

Comment:
version service is for component versions, we don't do anything with operator version

```suggestion
func (r *PerconaServerMySQLReconciler) reconcileVersions(ctx context.Context, cr *apiv1alpha1.PerconaServerMySQL) error {
```

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/151/commits/3336703af098a0467efa581d3f03f9a2de4a4c82